### PR TITLE
Cs/update pages titles

### PR DESCRIPTION
**Pull request by Charl1996**
## Product Description
Adds titles to some pages that didn't have or didn't show. 

([This slack](https://dimagi.slack.com/archives/C02QG63MU/p1663139964679989) chat can also be reviewed for reference)

## Technical Summary
It was noticed that there exist a few discrepancies between pages when navigating to different pages under the Data tab. Some pages showed titles and others not. Furthermore, between the pages that did have a title, different styles was observed. 

This PR does 2 things:
1) Adds the `lead` class to the `page_title` block
2) Use the `page_title` block on some pages so that their respective titles actually show.

Note: this PR only "fix" _some_ pages, but chances are that there's a bunch more that need changing/conforming to use the `page_title` block as the standard and not the "to-each-his-own" philosophy.  

Some before-and-after screenshots:

Before:
image

After:
image

One instance I do want to highlight is the following page, which I'm not sure what to do with the second title's ("Upload Lookup Tables") styling:
image

## Feature Flag
Not a specific FF

### Safety story
Tested locally

### Automated test coverage
Automated testing not necessary

### QA Plan
Only tested locally as this is styling

### Migrations
No migrations

### Rollback instructions
- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/export/views/list.py [hunk 1]
```diff
@@ -497,6 +497,7 @@ class BaseExportListView(BaseProjectDataView):
         for use in third-party data analysis tools.
     '''))
     is_odata = False
+    page_title = "Export Form Data"
```
Comment (orangejenny) [inline corehq/apps/export/views/list.py:500]: translate

**Review by orangejenny — APPROVED** [review]
looks good to me

**Comment by orangejenny** [comment]
On the lookup tables page, my first thought is to split it into separate pages. But I don't want to increase the scope of this work. As-is, the changes to that page are still an improvement.

**Review by biyeun — APPROVED** [review]
thank you! 🎉

**Pull request merged**